meecrowave-maven-plugin/src/main/java/org/apache/meecrowave/maven/MeecrowaveBundleMojo.java

```diff
@@ -57,7 +58,11 @@
 
 @Mojo(name = "bundle", requiresDependencyResolution = RUNTIME_PLUS_SYSTEM)
 public class MeecrowaveBundleMojo extends AbstractMojo {
-    private static final String DELETE_TEXT = "Just there to not loose the folder cause it is empty, you can safely delete.";
+    private static final String ZIP = "zip";
```

Comment:
don't think we need to duplicate the constants here (one for the annot and one for the class which is not used once the plugging passed but mvn can still be running (:run)), also same comment as for Meecrowave: not needed at runtime (so useless to keep) and not sure about readability